client/src/output.css

```diff
@@ -0,0 +1,6180 @@
+*, ::before, ::after {
```

Comment:
Consider not checking this (and other easily regenerated files). It bloats your diffs in git history.

Reply:
I struggled with adding this to my gitignore for quite a while this morning and eventually dropped it in favor of asking for help--I'd love to make this a focus of our meeting on Thursday.